src/seedu/addressbook/data/person/Address.java

```diff
@@ -45,7 +63,8 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
+                        && this.value.equals(((Address) other).value)); // state
+                                                                        // check
```

Comment:
Why is the comment occupying the entire line?
If I'm not mistaken it is caused by eclipse auto indent feature, please be careful and look through for these blind spots.